### DISCUSSION TITLE
Loading data only for the current date

### DISCUSSION THREAD

**Discussion by stwebage** [vkurko/calendar#225] [Q&A]
Hi, perhaps this question has already been asked, but I could not find it.
Is it possible to load data only for the current date? Let’s say I have a calendar for January 25, 2024 and I load data into the calendar only for this day.
If I selected the week from January 22 to January 28, 2024, then the data is loaded only from this interval.
Now I have a problem that I have downloaded 10,000 test posts and the system is starting to slow down, but I will have much more of them on the production server.

**Comment by vkurko** [comment]
Sure, it is possible. Please take a look at the [eventSources](https://github.com/vkurko/calendar?tab=readme-ov-file#eventsources) option.

**Comment by stwebage** [comment]
Good afternoon, if you don’t mind, can you show an example in what form the data for a response should be generated, I tried sending it in an array and json, but it is not displayed in the calendar

**Comment by vkurko** [comment]
Here is an [example](https://stackblitz.com/edit/vitejs-vite-nqrxzs?file=src%2FApp.svelte). The fetched data is in `public/events.json`.
